### DISCUSSION TITLE
How to change the font in FormulaControl?

### DISCUSSION THREAD

**Discussion by dekkab** [ForNeVeR/wpf-math#331] [Q&A] [answered]
I don't quite understand how you can change the font inside the "FormulaControl".

Are there any simple ways?

P.S. I tried using "FontStyle", but it didn't change the control visually in any way

**Comment by ForNeVeR** [comment, marked as answer]
There are two sets of fonts used by WPF-Math.

Main formula text uses Computer Modern, and this is impossible to change right now (because the font should have certain special features and known metrics for us to work with, and any random `.ttf` file doesn't have this information). There are plans to provide some alternatives, but you'll have to provide MetaFont information about them most likely. The root issue is now #332.

`\text{…}` inclusions use any installed system font; it is `Arial` by default, but easy to change using `FormulaControl::SystemTextFontName` property.
